### PR TITLE
Spin Up Hub: Redirect app

### PR DESCRIPTION
**Pull request by mikkelhegn**
A new Spin Up Hub submission to showcase a redirect app built using Spin.

### CODE REVIEW

**Comment by sohanmaheshwar** [comment]
`spin build` works fine. 

When I do `spin up`, I see the following error:

```
sohan@Sohans-MBP redirect % spin up
Logging component stdio to ".spin/logs/"
Storing default key-value data to ".spin/sqlite_key_value.db"
Error: Failed to instantiate component 'api'

Caused by:
    0: decoding custom section component-type:reactor
    1: component-type version 3 does not match supported version 2
Error: exit status: 1
```
Am I missing something in setup? wasn't able to debug

**Comment by mikkelhegn** [comment]
Hmm - Probably need to involve Spin engineers. I'm using v1.4.1 of Spin, but I believe the app still uses 1.0.0 SDKs. Let me open an issue in the Spin repo.

**Comment by mikkelhegn** [comment]
@sohanmaheshwar - FYI, I reverted the ADK to v1.0.0. I don't need 1.4.1 in the app (at least not yet) :-)

File: content/api/hub/sample_redirect.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+created_at = "2023-08-14T00:00:00Z"
+last_updated = "2023-08-14T00:00:00Z"
+spin_version = ">v1.0"
+summary =  "A sample of using mulitple components to create an app"
+url = "https://github.com/mikkelhegn/redirect"
+keywords = "rust, static content, components, html"
+
+---
+
+A nice litte app to create shortlinks, and generate QR codes.
```
Comment (jpflueger) [inline content/api/hub/sample_redirect.md:21]:
```suggestion
A nice little app to create short links and generate QR codes.
```

**Review by jpflueger — APPROVED** [review]
LGTM!

**Comment by tpmccallum** [comment]
Thanks @jpflueger

**Pull request merged**